iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -4704,11 +4746,13 @@
 			baseConfigurationReference = 347526FB1DC0B00F00918CF5 /* common-release.xcconfig */;
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
+				CLANG_ENABLE_MODULES = YES;
```

Comment:
It should be easy to remove some lines when amending the commit using Fork, Github App or other Git app with UI.